### DISCUSSION TITLE
Simple question/ feature request

### DISCUSSION THREAD

**Discussion by Fanat1cFire** [Fedoraware/Fedoraware#1194] [Ideas]
Does fedoraware have anti-obs? I know it has clean screenshot so the needed code is there but when I game cap with obs it still shows all Visuals.

If this is not a feature could it be added?

**Comment by ghost** [comment]
anti obs puts the visuals into an overlay and obs has the ability to not capture overlays, clean screenshot disables visual features while you are taking a screenshot. it will likely not be added.
